creator-node/src/redis.js

```diff
@@ -121,36 +121,36 @@ const WalletWriteLock = {
 
```

Comment:
changes here:
- function name
- exported as `redis` property instead of top level
- function params (removed logger)
- function is now async
- function now returns number of deleted keys
- function throws on internal error